src/App.tsx

```diff
@@ -10,33 +11,19 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, SetSelectedTab] = useState(tabs[0]);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${tabs.find(tab => (
+          selectedTab.id === tab.id))?.title}`}
```

Comment:
I think you can write like `{selectedTab.title}` without find method here

Reply:
Yeah, looks like it is overcomplicated here